### PR TITLE
Update PRIVACY.md

### PR DESCRIPTION
**Pull request by LandonLloyd**
# Pull Request

## Description:
I added some more information to the PRIVACY.md and also added some more helpful links.
 

## Related Issue:
This helps add to the PRIVACY.md

Closes #XXXXX

### CODE REVIEW

File: PRIVACY.md [hunk 1]
```diff
@@ -16,4 +16,8 @@ At any time, you have the right to request to view the Data pertaining to your D
 
 ## Underage Users
 
-The use of this Bot is not permitted for minors under the age of 13, or under the age of legal consent for their country. This is in compliance with the Discord Terms of Service.
+The use of this Bot is not permitted for minors under the age of 13, or under the age of legal consent for their country. This is in compliance with the [Discord Terms of Service.](https://discord.com/terms) No information will be knowingly stored from an underage user. If it is found out that the user is underage we will take all necessary action to delete the stored data.
```
Comment (nhcarrigan) [inline PRIVACY.md:19]:
```suggestion
The use of this Bot is not permitted for minors under the age of 13, or under the age of legal consent for their country. This is in compliance with the [Discord Terms of Service.](https://discord.com/terms). No information will be knowingly stored from an underage user. If it is found out that a user is underage we will take all necessary action to delete the stored data.
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
One quick fix.

**Comment by LandonLloyd** [comment]
Awesome I just commited them.

**Review by nhcarrigan — APPROVED** [review]
LGTM~! 🥳

**Pull request merged**